src/reynir_correct/config/GreynirCorrect.conf

```diff
@@ -3639,6 +3776,227 @@ Litháenskur
 kanada
 kanadamaður
 Kanadískur
+spánn
+Spænskur
+japan
+Japanskur
+
+# Language families
+# Unable to include: afróasískur, aímarískur, algískur, altískur, ástró-asískur, ástronesískur, dravidískur, dravídamál, eskimó-aleútískur, karíbamál, nígerkongómál, nílósaharamál, sinó-tíbetskur, suðurkákasískur, túpímál
```

Comment:
Missing from BÍN?

Reply:
Yes, they're all missing from BÍN and probably not very common anyway